hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerDoubleBuffer.java

```diff
@@ -440,10 +440,10 @@ private String addToBatch(Queue<DoubleBufferEntry<OMClientResponse>> buffer,
       // 1. It is first element in the response,
       // 2. Current request is createSnapshot request.
       // 3. Previous request was createSnapshot request.
-      if (response.isEmpty() ||
-          omResponse.getCreateSnapshotResponse() != null ||
-          (previousOmResponse != null &&
-              previousOmResponse.getCreateSnapshotResponse() != null)) {
+      if (response.isEmpty() || omResponse.getCreateSnapshotResponse()
```

Comment:
I made this change after the IDE warned me that `omResponse.getCreateSnapshotResponse() != null`  would always return true. Since it is coming from proto object `OMResponse` I guess [that would be the case.](https://developers.google.com/protocol-buffers/docs/reference/java-generated)

> Note that no Java protocol buffer methods accept or return nulls unless otherwise specified.

Reply:
> I think you can use omResponse.hasCreateSnapshotResponse(). That should remove all the confusion and concern.

Done. using hasCreateSnapshotResponse() now.